### ISSUE TITLE
discuss different logics for rotation

### ISSUE THREAD

**Issue by maxheld83** [maxheld83/pensieve#441]
![rotations](https://user-images.githubusercontent.com/5372770/47483630-077aef00-d83a-11e8-965a-13dc807de9bb.jpg)

**Comment by jfou** [comment]
Nice! Thanks, Max.

Any thoughts in including the logic of oblique rotations like centriod for when the statements have clear correlations? (sorry to bring that back, I'm just following your excellent progress!)

Jeremy


---------- Original Message ----------

**Comment by maxheld83** [comment]
Hi Jeremy, sorry I'm not sure I fully get the question.
Oblique rotations are also on the roadmap, but are probably a ways off (they make interpretation and some of the customarily used plots *really* difficult).

**Comment by jfou** [comment]
Hi Max,

I'm sorry I wasn't clear. I was asking if the logic of oblique rotations was going to be included since they are very useful in certain situations, such as when statements have inherent correlations and/or overlap.

As you point out, those rotations are more difficult and complicated, so I'm glad they remain on the roadmap. Great progress on the Pensieve framework so far!

Jeremy


---------- Original Message ----------

**Comment by maxheld83** [comment]
definitely! I'm tracking support for oblique rotations in https://github.com/maxheld83/pensieve/issues/442
